.eslintrc.json

```diff
@@ -1,16 +1,21 @@
 {
   "root": true,
-  "ignorePatterns": ["projects/**/*"],
+  "ignorePatterns": [
+    "projects/**/*"
```

Comment:
Looks like prettier wasn't run on that file :thinking:

Reply:
I did run it, but it changed the endOfLine character for pretty much every file. I've updated the prettier config therefore to use CRLF as endOfLine characters and run it afterwards. In the latest change, prettier only updated some files that I touched.